addons/crm/tests/test_crm_lead.py

```diff
@@ -676,7 +694,51 @@ def test_message_get_suggested_recipients(self):
             'lang_id': lang_en.id,
         })
         data = lead2._message_get_suggested_recipients()[lead2.id]
-        self.assertEqual(data, [(False, self.test_email, "en_US", 'Customer Email')])
+        self.assertEqual(len(data), 1)
+        self.assertEqual(data[0][:-1], (False, self.test_email, "en_US", 'Customer Email'))
+        default_create_values = data[0][-1]
+        self.assertEqual(default_create_values['lang'], lang_en.code)
+
+    @users('user_sales_manager')
+    def test_message_recipient_partner_auto_creation(self):
+        """Check default creates value for auto creation of recipient (customer)."""
+        ResPartner = self.env['res.partner']
+        for partner_name, name, email in [
```

Comment:
picky std: use tuple when list not needed

Reply:
picky tde: Hagrid !